src/async_fuse/memfs/s3_node.rs

```diff
@@ -155,7 +155,11 @@ impl<S: S3BackEnd + Send + Sync + 'static> S3Node<S> {
                 for (name, serial_dir_entry) in dir {
                     let child_ino = serial_dir_entry.get_child_ino();
                     // Fetch the child node from kv
-                    let child_node = meta.get_node_from_kv_engine(child_ino).await;
+                    // TODO: Handle this Error
```

Comment:
Please explain the reason to handle this later.

Reply:
I am sorry that this is my negligence, and it's fixed now.